src/validations/styleguides/xspec.sch

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<?xml-model schematypens="http://purl.oclc.org/dsdl/schematron" href="sch.sch" ?>
+<?xml-model schematypens="http://purl.oclc.org/dsdl/schematron" href="../styleguides/sch.sch" ?>
```

Comment:
Why is `../styleguides` necessary if the document is in the same directory?

Reply:
Corrected in bcec36e.